tests/tests_async.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf8 -*-
+import datetime
```

Comment:
Nit: We can probably find an unserializable object without needing to import another library.  For example, `object()` isn't serializable 🙃

Reply:
@javulticat no problem, updated!